.github/workflows/ci.yaml

```diff
@@ -1,6 +1,6 @@
 name: Node.js CI
 
-on: [push, pull_request]
+on: push
```

Comment:
It was double-running the tests before, since GHA doesn't treat on things are `push AND pull_request` it's actually `push OR pull_request` and it just runs it for each true condition.